package.json

```diff
@@ -60,5 +60,8 @@
     "lint-check": "yarn workspaces run lint-check",
     "test": "yarn workspaces run test",
     "build": "yarn workspaces run build"
+  },
+  "dependencies": {
+    "@agoric/make-promise": "0.0.3"
```

Comment:
I don't think this belongs in the top-level `package.json`, unless we have build scripts that use it (and even then it would be in `devDependencies`). Did something not work without this?

Reply:
I don't usually make changes like this without a prompt, but it seems to be fine without it, so I've removed it.